src/App.js

```diff
@@ -27,6 +27,7 @@ export const App = () => {
   return (
     <div className={styles.app}>
       <Header />
+      <Footer />
```

Comment:
another good solution that does not include using the z-index attribute is to use the function `isMobileOrTablet` and use it instead of using `isMobile` 

`import React from 'react';
import useBreakpoint from 'use-breakpoint';

import {Breakpoint, isMobileOrTablet} from '../../../enums';
import {useFooterTranslation} from '../../../hooks';
import styles from './Footer.module.scss';

export const Footer = () => {
  const {rightsTxt} = useFooterTranslation();
  const {breakpoint} = useBreakpoint(Breakpoint);
  return !isMobileOrTablet(breakpoint) ? (
    <footer className={styles.footer}>
      <div className={styles.copyright}>{rightsTxt}</div>
    </footer>
  ) : null;
};
`

Reply:
I would prefer not to change the current behavior just for a bug fix unless the product wants it